### PR TITLE
Updated the NOTES for istio.

### PR DESCRIPTION
**Pull request by gyliu513**
/cc @lachie83

### CODE REVIEW

**Comment by gyliu513** [comment]
ping @lachie83

File: incubator/istio/templates/NOTES.txt [hunk 1]
```diff
@@ -56,7 +56,13 @@ Verifying the ServiceGraph service
 
 Deploy your App!
 
-  kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
+  If you have Istio Initializer installed, run the following to deploy your App:
```
Comment (lachie83) [inline incubator/istio/templates/NOTES.txt]: Please conditionalize and print the correct command eg. `{{ if .Values. initializer.enabled }}`

**Comment by gyliu513** [comment]
@lachie83 sorry for the late update, it is done now.

File: incubator/istio/templates/NOTES.txt (hunk 1 above)
Comment (gyliu513) [reply incubator/istio/templates/NOTES.txt]: Done

**Comment by gyliu513** [comment]
```
root@k8s001:~/go/src/github.com/gyliu513/charts# helm lint incubator/istio/
==> Linting incubator/istio/
Lint OK

1 chart(s) linted, no failures
```

Done, @lachie83

**Comment by mattfarina** [comment]
@gyliu513 Same issue I noted on #2439. Given that you have two PRs against the same chart and they are both doc related you might want to combine them into one doc PR. Or, you'll need to handle the version bumping so each does it right.

**Comment by gyliu513** [comment]
Thanks @mattfarina for the explanation, I recalled that there are indeed exceptions for this, such as https://github.com/kubernetes/charts/pull/2059 , let's get some comments from @lachie83 as well ;-)

**Comment by lachie83** [comment]
I'm okay with these going in without a version bump

**Pull request merged**